core/src/test/java/eu/kevin/core/enums/KevinCountryTest.kt

```diff
@@ -0,0 +1,40 @@
+package eu.kevin.core.enums
+
+import eu.kevin.testcore.base.BaseUnitTest
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import org.junit.Assert
+import org.junit.Test
+
+@ExperimentalCoroutinesApi
+class KevinCountryTest : BaseUnitTest() {
+
+  @Test
```

Comment:
Another thing which I have noticed, please update indentation from 2 spaces to 4, because this format is used across the project.

Reply:
You are right! 👍 Thank you!